docs/get-started/getting-started-on-prem.md

```diff
@@ -6,13 +6,13 @@ parent: Getting-Started
 ---
 
 # Getting Started with RAPIDS Accelerator with on premise cluster or local mode
-## Spark Deployment Methods
+## The primary methods of deploy Spark
```

Comment:
Although I know this was suggested in issue https://github.com/NVIDIA/spark-rapids/issues/8280 , I think the original phrasing is ok
```suggestion
## Spark Deployment Methods
```